Dev_Board/Dev_Board.kicad_pcb

```diff
@@ -0,0 +1,6563 @@
+(kicad_pcb (version 20221018) (generator pcbnew)
```

Comment:
Make sure you're following JLCPCB's manufacturing capabilities. You can check these on the website, and I've went ahead and added them to your board & made it compliant to their capabilities.